`greenmail` exception after mvn execution finishes

At the end of a `mvn install` that succeeded, the following exception was printed to the console: 

```txt
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  58.684 s
[INFO] Finished at: 2025-06-18T11:15:04+02:00
[INFO] ------------------------------------------------------------------------
Exception in thread "Thread-1" java.lang.NoClassDefFoundError: com/icegreen/greenmail/foedus/util/Workspace
	at com.icegreen.greenmail.util.GreenMail.<init>(GreenMail.java:60)
	at com.btmatthews.maven.plugins.emailserver.greenmail.GreenmailMailServer.start(GreenmailMailServer.java:47)
	at com.btmatthews.utils.monitor.Monitor.runMonitor(Monitor.java:85)
	at com.btmatthews.utils.monitor.Monitor$1.run(Monitor.java:108)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.ClassNotFoundException: com.icegreen.greenmail.foedus.util.Workspace
	at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
	... 5 more
```

The exit code of the mvn command still was 0.

I'm told this is related to some test-email-server that does not gracefully shutdown

--- a/jqm-all/jqm-integration-tests/pom.xml
+++ b/jqm-all/jqm-integration-tests/pom.xml
@@ -142,6 +142,14 @@
                     <monitorPort>11025</monitorPort>
                 </configuration>
 
+                <dependencies>
+                    <dependency>
+                        <groupId>com.icegreen</groupId>
+                        <artifactId>greenmail</artifactId>
+                        <version>1.3.1b</version>
+                    </dependency>
+                </dependencies>
+
                 <executions>
                     <execution>
                         <id>run-mail</id>

